opm-test.sh

```diff
@@ -1,26 +1,58 @@
 #!/bin/bash
```

Comment:
could we make this a go file that `exec`s opm (or just exercises the commands)?

Reply:
that will also let you use build tags to selectively build for os

Reply:
sure!